docs/tutorials/hello-minikube/hello-minikube.md

```diff
@@ -0,0 +1,305 @@
+on OS X. If you are on a different platform like Linux, or using VirtualBox
+instead of Docker for Mac, the instructions to install Minikube may be
+slightly different. For general Minikube installation instructions, see
+the [Minikube installation guide](docs/getting-started-guides/minikube/).
+
+### Download and install the Minikube release
+
+Use `curl` to download and install the latest Minikube release:
+
+```shell
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-darwin-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+```
+
+Use Homebrew to install the xhyve driver and set its permissions.
+
+```shell
+brew install docker-machine-driver-xhyve
+sudo chown root:wheel $(brew --prefix)/opt/docker-machine-driver-xhyve/bin/docker-machine-driver-xhyve
+sudo chmod u+s $(brew --prefix)/opt/docker-machine-driver-xhyve/bin/docker-machine-driver-xhyve
+```
+
+Download the latest version of the `kubectl` command-line tool, which you can
+use to interact with Kubernetes clusters.
+
+```shell
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl
+chmod +x ./kubectl
+sudo mv ./kubectl /usr/local/bin/kubectl
+```
+
+Start the Minikube cluster:
+
+```shell
+minikube start --vm-driver=xhyve
+```
+
+The `--vm-driver=xyhve` flag specifies that we are using Docker for Mac. The
+default VM driver is VirtualBox. Now set the Minikube context:
+
+```shell
+kubectl config use-context minikube
+```
+
+Now you should be able to run:
+
+```shell
+kubectl cluster-info
+```
+
+to verify the cluster is running and view its metadata.
+
+## Create your Node.js application
+
+The next step is to write the application. Save this code in a folder called "`hellonode/`" with the filename `server.js`:
+
+#### server.js
+
+{% include code.html language="js" file="server.js" ghlink="/docs/tutorials/hello-minikube/server.js" %}
+
+Now run this simple command:
+
+```shell
+node server.js
+```
+
+You should be able to see your "Hello World!" message at http://localhost:8080/. 
+
+Stop the running node server by pressing Ctrl-C.
+
+Now let’s package this application in a Docker container.
+
+## Create a Docker container image
+
+Next, create a file, also within `hellonode/` named `Dockerfile`. A Dockerfile describes the image that you want to build. Docker container images can extend from other existing images so for this image, we'll extend from an existing Node image.
+
+#### Dockerfile
+
+{% include code.html language="conf" file="Dockerfile" ghlink="/docs/tutorials/hello-minikube/Dockerfile" %}
+
+This "recipe" for the Docker image will start from the official Node.js LTS image found on the Docker registry, 
+expose port 8080, copy our `server.js` file to the image and start the Node server.
+
+Since this tutorial uses Minikube, instead of pushing Docker images to a
```

Comment:
I don't understand this paragraph. I think that's because I don't understand what the Minikube docker daemon is.